paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -2322,71 +2285,17 @@ class CustomerSheetViewModelTest {
         )
 
         viewModel.handleViewAction(
-            CustomerSheetViewAction.OnCollectBankAccountResult(
-                mockUSBankAccountResult(
-                    isVerified = true,
-                )
-            )
+            CustomerSheetViewAction.OnBankAccountSelectionChanged(mockUSBankAccountPaymentSelection())
         )
 
         viewModel.viewState.test {
             val viewState = awaitViewState<AddPaymentMethod>()
             assertThat(
-                viewState.shouldDisplayDismissConfirmationModal(
-                    isFinancialConnectionsAvailable = { true }
-                )
+                viewState.shouldDisplayDismissConfirmationModal()
             ).isTrue()
         }
     }
 
-    @Test
-    fun `When in add flow and unverified us bank account is retrieved, then shouldDisplayConfirmationDialog should be false`() = runTest(testDispatcher) {
```

Comment:
With the proposed behavior change (show the dialog regardless of bank account type), these tests aren’t necessary anymore.